src/js/touchevents.tsx

```diff
@@ -31,6 +31,10 @@ export type TouchEventBoundaryProps = {
    * React Node wrapped by TouchEventBoundary.
    */
   children?: React.ReactNode;
+  /**
+   * React Node wrapped by TouchEventBoundary.
```

Comment:
```suggestion
   * Label Name used to identify the touched element.
   * Default: `sentry-label`
   * Fallback: `element.displayName`
```

Reply:
Good catch. I fixed the first line of the description but I don't think the default and fallback values fit in here, because setting a `sentry-label` on a specific component will actually overwrite the `labelName` set via `TouchEventBoundary`.